exonum/src/blockchain/service.rs

```diff
@@ -187,9 +191,41 @@ impl ServiceContext {
             fork,
             stored_configuration,
             height,
+            logger,
         }
     }
 
+    /// Returns current logger context, usable when important extended logging system.
+    /// If one needs more logging feature, one can use `slog` ecosystem to print log message.
+    ///
+    /// Example:
+    /// ```rust, no_build
+    /// #[macro_use] extern crate slog;
+    ///
+    /// // context: ServiceContext
```

Comment:
I propose to use `let context: ServiceContext = ...` instead of comment, because this isn't going to be compiled anyway.